### PR TITLE
{AKS} `aks trustedaccess rolebinding create`: Remove `--source-resource-id` parameter alias `-r`

### PR DESCRIPTION
**Pull request by YitongFeng-git**
**Related command**
`az aks trustedaccess rolebinding create`

**Description**
 remove `az aks trustedaccess rolebinding create` command --source-resource-id parameter alias -r

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -2072,7 +2072,7 @@
         - name: --roles
           type: string
           short-summary: Specify the comma-separated roles.
-        - name: --source-resource-id -r
+        - name: --source-resource-id
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py:2075]: What are the considerations for removing this option?

As this is a breaking change, need to wait until the next breaking change window. Before that, you could mark it as deprecated.

Comment (YitongFeng-git) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:2075]: After discussed we decided to remove the alias '-r'. 

As TA command hasn't been released ever, we can make this change before this version's release

Comment (FumingZhang) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py:2075]: Agree, the option to be removed was introduced in #28171 on 01/12 and has not been released

**Review by FumingZhang — COMMENTED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85912668&view=results) to validate the change.

**Comment by FumingZhang** [comment]
Queued another [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85913949&view=results)

**Pull request merged**